xwiki-platform-core/xwiki-platform-activitystream/xwiki-platform-activitystream-ui/src/main/resources/Main/ActivityTranslations.ca.xml

```diff
@@ -20,24 +20,21 @@
  * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
 -->
 
-<xwikidoc reference="Main.ActivityTranslations" locale="ca">
+<xwikidoc version="1.3" reference="Main.ActivityTranslations" locale="ca">
   <web>Main</web>
   <name>ActivityTranslations</name>
   <language>ca</language>
   <defaultLanguage>en</defaultLanguage>
   <translation>1</translation>
-  <parent>Main.Activity</parent>
   <creator>xwiki:XWiki.Admin</creator>
+  <creationDate>1390437642000</creationDate>
+  <parent>Main.Activity</parent>
   <author>xwiki:XWiki.Admin</author>
-  <customClass/>
   <contentAuthor>xwiki:XWiki.Admin</contentAuthor>
-  <creationDate>1390416042000</creationDate>
-  <date>1390416053000</date>
-  <contentUpdateDate>1390416047000</contentUpdateDate>
+  <date>1512001733000</date>
+  <contentUpdateDate>1512001733000</contentUpdateDate>
   <version>1.1</version>
   <title/>
-  <defaultTemplate/>
-  <validationScript/>
   <comment/>
   <minorEdit>false</minorEdit>
   <syntaxId>plain/1.0</syntaxId>
```

Comment:
Note that you are updating a file that is automatically generated by our http://l10n.xwiki.org translation tool, which is not a good idea, since you risk losing your changes.